cmd/web/tmpl/create.tmpl

```diff
@@ -9,5 +9,18 @@
 
 <br />
 <br />
+
+{{if len .RecentPastes | eq 0}}
```

Comment:
Note for reviewers: go templates have the concept of expressions, but they call them pipelines. This pipeline is the equivalent of doing the following in c style languages:

```go
if eq(len(.RecentPastes), 0)
```

https://pkg.go.dev/text/template#hdr-Pipelines